cogs/sfu/roads.py

```diff
@@ -131,11 +131,14 @@ async def cam(self, ctx: Context, cam: str = ""):
             fetchedData = self.httpRetrySession.get(camera, headers=self.headers)
             fetchedData.raise_for_status()
         except requests.exceptions.HTTPError:
-            await ctx.send(":warning: This webcam is currently unavailable!")
+            await ctx.send(
+                ":warning: This webcam is currently unavailable! Please try again later"
```

Comment:
micronit: end the sentence with a period like line 140.
```suggestion
                ":warning: This webcam is currently unavailable! Please try again later."
```

Reply:
Addressed in latest diff